h2o-core/src/main/java/water/api/schemas3/RapidsHelpV3.java

```diff
@@ -4,27 +4,22 @@
 import water.api.API;
 
 /**
+ * Help for the rapids language
  */
 public class RapidsHelpV3 extends SchemaV3<Iced, RapidsHelpV3> {
 
-  @API(help="Rapids language, organized in a form of a tree (so that similar constructs are grouped together.",
+  @API(help="Description of the rapids language.",
       direction=API.Direction.OUTPUT)
-  public RapidsExpressionV3 syntax;
+  public RapidsExpressionV3[] expressions;
 
   public static class RapidsExpressionV3 extends SchemaV3<Iced, RapidsExpressionV3> {
     @API(help="(Class) name of the language construct")
     public String name;
 
-    @API(help="If true, then this is not a standalone construct but purely a grouping level.")
-    public boolean is_abstract;
-
     @API(help="Code fragment pattern.")
     public String pattern;
 
     @API(help="Description of the functionality provided by this language construct.")
     public String description;
-
-    @API(help="List of language constructs that grouped under this one.")
```

Comment:
If you agree, I would leave this up on the possible future changes/improvements of the rapids help. For now it at least works and doesn't crash